src/gitflow/actions/ReInitRepoAction.java

```diff
@@ -0,0 +1,37 @@
+package gitflow.actions;
+
+import git4idea.commands.GitCommandResult;
+import git4idea.repo.GitRepository;
+import gitflow.GitflowInitOptions;
+
+public class ReInitRepoAction extends InitRepoAction {
+    ReInitRepoAction() {
+        super("Re-init Repo");
+    }
+
+    ReInitRepoAction(GitRepository repo) {
+        super(repo, "Re-init Repo");
+    }
+
+    @Override
+    protected String getPublishedFeatureMessage() {
```

Comment:
Can you please change this name to be more fitting? I realize that's my fault and not yours since it was like this in the original InitRepo, but it's a good opportunity to fix that :)